docs/source/reference/auth.rst

```diff
@@ -247,6 +283,13 @@ Use ``helm upgrade`` to redeploy the API server helm chart with the ``skypilot-o
       --set auth.oauth.client-secret=<CLIENT SECRET> \
       --set auth.oauth.email-domain=<EMAIL DOMAIN> # optional
 
+If you are using Microsoft Entra ID or any other provider that requires the redirect URI to be HTTPS, you will need to set one additional Helm value:
```

Comment:
Minor: Wondering if it is possible to provide a tab above for an alternative way for specifying the values in a `values.yaml` to be used for upgrade, so that people don't need to specify different elements in command line.